apps/src/templates/census2017/CensusInaccuracyReview.jsx

```diff
@@ -0,0 +1,400 @@
+                <td>
+                  {this.state.toReview.how_many_10_hours}
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  How many 20 hours
+                </td>
+                <td>
+                  {this.state.toReview.how_many_20_hours}
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Block programming
+                </td>
+                <td>
+                  {this.state.toReview.topic_blocks ? 'Yes' : 'No'}
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  Text programming
+                </td>
+                <td>
+                  {this.state.toReview.topic_text ? 'Yes' : 'No'}
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <br />
+        <div>
+          <p style={{width: '75%'}}>
+            Explain why we should or should not override the data on this school to say
+            that they <b>{this.state.toReview.teaches_cs ? 'DO' : 'DO NOT'}</b> offer Computer Science:
+          </p>
+          <br />
+          <textarea type="text" style={{height: 100, width: '100%'}} onChange={(event) => {this.setState({notes: event.target.value});}} />
+          <br />
+          {this.state.notes && (
+             <div>
+               <Button
+                 onClick={this.submitInvestigationWithOverride}
+                 size="large"
+                 text={overrideText}
+               />
+               <br />
+               <Button
+                 onClick={this.submitInvestigationWithoutOverride}
+                 size="large"
+                 text="Resolve without override"
+                 style={{marginTop: 5}}
+               />
+             </div>
+          )}
+        </div>
+      </div>
+    );
+  };
+
+  submitInvestigation = (override) => {
+    const reportId = this.state.toReview.id;
+    $.ajax({
+      url: "/census/review",
+      type: "post",
+      dataType: "json",
+      data: {
+        authenticity_token: this.props.authenticityToken,
+        census_submission_id: reportId,
+        override: override,
+        notes: this.state.notes,
+      },
+    }).done(() => (this.handleSuccessfulInvestigationSubmittion(reportId))).fail(this.handleInvestigationSubmittionError);
+  };
+
+  submitInvestigationWithOverride = () => {
+    this.submitInvestigation(this.state.toReview.teaches_cs ? 'Y' : 'N');
+  };
+
+  submitInvestigationWithoutOverride = () => {
+    this.submitInvestigation(null);
+  };
+
+  handleSuccessfulInvestigationSubmittion = (reportId) => {
+    this.setState((prevState) => ({
+      resolvedReports: prevState.resolvedReports.concat([reportId]),
+      toReview: undefined,
+      notes: ''
+    }));
+    $('#report-header')[0].scrollIntoView();
+  };
+
+  handleInvestigationSubmittionError = (error) => {
+    console.log(error.responseText);
```

Comment:
should this be `console.warn` or `console.error` instead of log? Do we want to display the error to the user in any other way, in the UI?

Reply:
Added display for the error case.